documentation/agent/usage.md

```diff
@@ -35,27 +58,47 @@ by sending HTTP requests to the agent.
 
 Example:
 
-    $> fusioninventory-agent --server http://my.server/plugins/fusioninventory/ --daemon
+    $> fusioninventory-agent --server http://my.server --daemon
+
+That's the easiest mode to set up, offering (almost) complete control from a
+centralized point, fully compatible with all available agent features, and the
+most flexible in term of usage. However, that's also the most expensive in term
+of resources usage, and the less secure. And the more important: who controls
+the GLPI servers also controls all assets where an agent is installed, with
+ability to execute arbitrary code with full system privileges, which is after
```

Comment:
Again this should be discussed in a dedicated paragraph, the ability to execute arbitrary code with full system privileges only exists if Deploy task is installed with the and AND is enabled by configuration.

Reply:
I just isolated this sentence in a specific paragraph, with a distinction with "executing code" (always true) and "executing arbitrary commands" (in specific case). It could eventually get promoted to a full section somewhere in the plan, but I thinks its current included situation make it easier to understand than a reference to somewhere else. Feel free to propose an alternative, however.